staramr/databases/resistance/pointfinder/ARGDrugTablePointfinder.py

```diff
@@ -34,7 +34,7 @@ def get_drug(self, organism, gene, position):
                 table['Codon Pos.'] == position)]['Drug']
         if (drug.empty):
             logger.warning(
-                "No drug found for organism=" + organism + ", gene=" + gene + ", position=" + position)
+                "No drug found for organism=" + organism + ", gene=" + gene + ", position=" + str(position))
```

Comment:
This might make your logging a little easier:
> To log variable data, use a format string for the event description message and append the variable data as arguments.
 - https://docs.python.org/3/howto/logging.html#logging-variable-data

e.g.
```python
logger.warning("No drug found for organism=%s, gene=%s, position=%s", organism, gene,  position)
```

Reply:
Thanks. I've changed all logging statements to use string formatting e9fe4b19884197ab169c35c1e0a58d0f67f3c3f0

Reply:
Looks good!

I might be wrong, but I think you can remove the `str()` and `repr()` calls if you're using the `"%s"` string formatting style just like with `str.format()`.

Reply:
You're right. Thanks. Fixed in f933fd6458281f385c0a0b9c4a5d612310830934